### PR TITLE
Improved Formatting for Tips

### PR DESCRIPTION
**Pull request by rajmanikush**
Tips were being merged with the contents so I have formatted it to look uniform throughout the read

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ Turn off the autopilot and take control. Constantly critique and appraise your w
 ## 1.-The Cat Ate My Source Code
 **Tip 3: Provide Options, Don't Make Lame Excuses**
 
-Instead of excuses, provide options. Don't say it can't be done; explain what can be done to salvage the situation.
```
Comment (HugoMatilla) [inline readme.md]: I will keep as is now.

File: readme.md [hunk 2]
```diff
@@ -333,7 +335,7 @@ The refinement and confidence in the schedule gets better and better each iterat
 **Tip 19: Iterate the Schedule with the Code**
 
 ### What to Say When Asked for an Estimate
-**"I'll get back to you."**
```
Comment (HugoMatilla) [inline readme.md]: I will keep the "". Makes it easier to see that is a sentence to say.

**Review by HugoMatilla — CHANGES_REQUESTED** [review]
Thank you for the PR. If you change what I mentioned in the comments I will be glad to merge the PR.

File: readme.md (hunk 1 above)
Comment (rajmanikush) [reply readme.md]: sure, let me update it

File: readme.md (hunk 2 above)
Comment (rajmanikush) [reply readme.md]: sure!

**Comment by HugoMatilla** [comment]
Thank's 💯

**Pull request merged**